README.md

```diff
@@ -38,6 +38,7 @@ This extension requires you to provide an OAuth token, to create it go [here](ht
   "githubNotificationsBell.color": "", // Bell's color when there are some notifications
   "githubNotificationsBell.hideIfNone": true, // Hide the bell if there are no notifications
   "githubNotificationsBell.showNumberOfNotifications": true // Show the number of notifications alongside the bell icon
+  "githubNotificationsBell.baseUrl": "https://github.com" // The Github URL to query against. Github Enterprise may use a non-standard url
```

Comment:
Would it be ok to name this `githubNotificationsBell.domain` and expect a value such as `github.com`? Or would that be confusing or people run the enterprise version of GitHub over `http`?

Reply:
_Hopefully_ noone is running GitHub over `http`, so I _think_ dropping the protocol and forcing https should be a safe enough change.

Will rename the variable too - wasn't mad on baseUrl anyway :)